gertjvr/serverless-plugin-simulate
# Mocking supporting services with docker-compose

## What did you implement:

Closes #1 #28 #29

## How did you implement it:

Added configuration to start supporting services, by running a docker-compose file.

```yml
custom:
  simulate:
    services: docker-compose.yml
```

## How can we verify it:

Create a docker-compose file in the same location as serverless.yml and run the simulate.
run `docker-compose ps` in a terminate and you should see all the service started in your `docker-compose.yml`

## Todos:



***Is this ready for review?:*** YES
